### PR TITLE
Upgrading rs to 0.34.2

### PR DESCRIPTION
**Pull request by Bidek56**
Upgrading rs to 0.34.2
Upgrading Biome to 1.3.3

### CODE REVIEW

**Comment by Bidek56** [comment]
@universalmind303 It is building fine using `version = "0.34.2"` on my laptop. 
Any idea why CI is pulling `main` instead of `version = "0.34.2"` ? Thx

File: Cargo.toml [hunk 1]
```diff
@@ -17,21 +17,23 @@ ahash = "0.8.3"
...
 napi-derive = {version = "2.13.0", default-features = false}
-polars-core = {git = "https://github.com/pola-rs/polars.git", rev = "ec0c91f93fcd1ac355c667d6c3c3f30b257ea0a6", default-features = false}
-polars-io = {git = "https://github.com/pola-rs/polars.git", rev = "ec0c91f93fcd1ac355c667d6c3c3f30b257ea0a6", default-features = false}
-polars-lazy = {git = "https://github.com/pola-rs/polars.git", rev = "ec0c91f93fcd1ac355c667d6c3c3f30b257ea0a6", default-features = false}
-polars-ops = {git = "https://github.com/pola-rs/polars.git", rev = "ec0c91f93fcd1ac355c667d6c3c3f30b257ea0a6", default-features = false}
-thiserror = "1.0.47"
+polars-core = {git = "https://github.com/pola-rs/polars.git", version = "0.34.2", default-features = false}
+polars-io = {git = "https://github.com/pola-rs/polars.git", version = "0.34.2", default-features = false}
+polars-lazy = {git = "https://github.com/pola-rs/polars.git", version = "0.34.2", default-features = false}
+polars-ops = {git = "https://github.com/pola-rs/polars.git", version = "0.34.2", default-features = false}
```
Comment (universalmind303) [inline Cargo.toml:23]:
```suggestion
polars-core = {version = "0.34.2", default-features = false}
polars-io = {version = "0.34.2", default-features = false}
polars-lazy = {version = "0.34.2", default-features = false}
polars-ops = {version = "0.34.2", default-features = false}
```

File: Cargo.toml [hunk 2]
```diff
@@ -92,9 +93,11 @@ features = [
   "list_eval",
   "arg_where",
   "timezones",
+  "string_pad",
+  "peaks",
 ]
 git = "https://github.com/pola-rs/polars.git"
-rev = "ec0c91f93fcd1ac355c667d6c3c3f30b257ea0a6"
+version = "0.34.2"
```
Comment (universalmind303) [inline Cargo.toml:100]: remove `git = ...`

File: Cargo.toml [hunk 3]
```diff
@@ -17,21 +17,23 @@ ahash = "0.8.3"
...
+polars-core = {git = "https://github.com/pola-rs/polars.git", version = "0.34.2", default-features = false}
+polars-io = {git = "https://github.com/pola-rs/polars.git", version = "0.34.2", default-features = false}
+polars-lazy = {git = "https://github.com/pola-rs/polars.git", version = "0.34.2", default-features = false}
+polars-ops = {git = "https://github.com/pola-rs/polars.git", version = "0.34.2", default-features = false}
+thiserror = "1"
 smartstring = { version = "1" }
 serde_json = {version = "1"}
 either = "1.9"
+rayon = "1.8"
+hashbrown = { version = "0.14", features = ["rayon", "ahash"] }
```
Comment (universalmind303) [inline Cargo.toml:29]: why did we need to add these?

Comment (Bidek56) [reply Cargo.toml:29]: I was getting an error hence added them, now they are not needed. I will remove them.

**Pull request closed without merging**